source/dub/compilers/utils.d

```diff
@@ -255,9 +255,9 @@ Path generatePlatformProbeFile()
 		fil.close();
 	}
 
-	// NOTE: This must be kept in sync with the dub.platform module
 	fil.write(q{
 		module dub_platform_probe;
+		import std.array;
```

Comment:
Importing phobos in the probe introduced a performance regression, see https://github.com/dlang/dub/pull/1318.